### DISCUSSION TITLE
[v2] How to render Nuxt components (e.g. nuxt-link, nuxt-img) in Markdown file with Content v2?

### DISCUSSION THREAD

**Discussion by nwittwer** [nuxt/content#1663] [Q&A]
Hi, a few questions regarding Markdown files & syntax w/ Content v2 (using v2.2.1) w/ Nuxt 3 (3.0.0-rc.13). 

Background: I've used @nuxt/content v1 in the past, so I have quite a few `.md` files where HTML components are used.

I noticed there was an answer regarding components with `props` specifically here: https://github.com/nuxt/content/discussions/1363#discussioncomment-3188439, but not sure if there's a similar answer to _all_ components as well.

**1. Is it still possible to reference components like `nuxt-link` and `nuxt-img` by using HTML tags?**
```md
Lorem ipsum dolor sit amet, consectetur adipiscing elit <nuxt-link to="/page/123">My Awesome Link</nuxt-link>
```

I am regularly seeing this error for both `NuxtImg` and `NuxtLink` when referencing them in a Markdown (.md) file like above^:
```
[Vue warn]: Failed to resolve component: NuxtImg
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement.
```

**2. If not, this means I have to change all HTML tags in my Markdown files over to the new MDC syntax?**

**3. Is there an example of how the MDC syntax would work for inserting a component which is inside of a multi-line block of text?**

For example, in @nuxt/content (v1) I used to be able to do this:
```
Lorem ipsum dolor sit amet, consectetur adipiscing elit <nuxt-link to="/page/123">My Awesome Link</nuxt-link>. Quisque dignissim eu dolor sit amet volutpat. Suspendisse eget porta massa, viverra euismod nunc. In quis lorem ut justo gravida tempor vel ut ante. Aenean vehicula id metus at venenatis. Maecenas rutrum sem at tincidunt feugiat <nuxt-link to="/page/345">My Other Link</nuxt-link>. 
```

How would this look in @nuxt/content (v2)? It would be nice to be able to continue having it all in one block/paragraph. From the current docs it seemed like the Props > Inline method example would be able to handle this, but maybe the syntax requires it as multiple lines?

I was expecting this to work based on the Docs examples, however it just shows up as plain text:
<pre><code>
Lorem ipsum dolor sit amet <b>::nuxt-link{to="/page/123"}Link Title::</b> consectetur adipiscing elit
</pre></code>

Thanks!

**Comment by Barbapapazes** [comment]
Hello,

Why use Nuxt Link and not the markdown link? Il fact, markdown link is converted into a nuxt link!

https://github.com/nuxt/content/blob/main/src/runtime/components/Prose/ProseA.vue
